web/portal/platform/src/translations/en.json

```diff
@@ -155,7 +155,7 @@
   "Residential Device_other": "Residential Devices",
   "Restore": "Restore",
   "Restore default template": "Restore default template",
-  "Restore default template?": "Restore default template?",
+  "Restore default template?": "¿Restore default template?",
```

Comment:
There is no open signs in english, remove ¿

Reply:
Also I don't see the spanish translations here, but I should be: _Restaurar plantilla por defecto_ for the buttons and _¿Restaurar plantilla por defecto?_ for the modal dialog